test/blackbox/utils/Utils.ts

```diff
@@ -0,0 +1,67 @@
+import {promises as fs} from 'fs';
+import * as path from 'path';
+import { AbstractGenerator, FormatHelpers, OutputModel } from '../../../src';
+import {promisify} from 'util';
+import {exec} from 'child_process';
+const promiseExec = promisify(exec);
+
+/**
+ * Exec a command and if any errors occur reject the promise.
+ * 
+ * @param absolutePathToFile 
+ * @param generator  
+ */
+ export async function generateModels(absolutePathToFile: string, generator: AbstractGenerator) {
+  const inputFileContent = await fs.readFile(absolutePathToFile);
+  const input = JSON.parse(String(inputFileContent));
+  return await generator.generate(input);
```

Comment:
If you have async function which returns some result from promise, then last `await` is unnecessary, because finally you return Promise. It's only a tip, `return await` is not a problem for me :)